include/dxc/Support/HLSLOptions.td

```diff
@@ -540,6 +540,8 @@ def all_resources_bound : Flag<["-", "/"], "all-resources-bound">, Flags<[CoreOp
   HelpText<"Enables agressive flattening">;
 def all_resources_bound_ : Flag<["-", "/"], "all_resources_bound">, Flags<[CoreOption, HelpHidden]>, Group<hlslcomp_Group>,
   HelpText<"Enables agressive flattening">;
+def skip_hash : Flag<["-", "/"], "skip-hash">, Flags<[CoreOption, HelpHidden]>, Group<hlslcomp_Group>,
```

Comment:
What's the use case for this?

Reply:
Just in case for someone affected by behavior change has a work around.

Reply:
We discussed offline - I'm keen to avoid adding new flags to the validator without any design review. I'd recommend removing the parts of this change around skipping hashes and filing an issue to follow up on it.

Reply:
Removed.